sechub-api-java/build.gradle

```diff
@@ -6,46 +6,34 @@
  * ============================================================================
  */
 
-ext {
-    jersey_version = "2.35"
-    jackson_version = "2.13.2"
-    jackson_databind_version = "2.13.2.2"
-    jackson_databind_nullable_version = "0.2.3"
+// see https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-gradle-plugin/README.adoc
+plugins {
+	id 'org.openapi.generator'          
 }
 
 dependencies {
-    // The production code uses the SLF4J logging API at compile time
-    implementation spring_boot_dependency.slf4j_api
-    implementation spring_boot_dependency.jackson_core
-    implementation spring_boot_dependency.jackson_databind
-    
     implementation project(':sechub-commons-model')
     
-    implementation library.apache_httpcomponents_client
-
+    implementation files('gen/build/libs/sechub-api-java-0.0.0.jar')
+    
+    testImplementation project(':sechub-testframework')
     testImplementation spring_boot_dependency.junit_jupiter
+}
 
-    testImplementation spring_boot_dependency.mockito_core
-    testImplementation library.mockito_inline
-    testImplementation spring_boot_dependency.hamcrest
+sourceSets {
+    main {
```

Comment:
This seems to be non necessary code.

If this is a preparation for the next steps we should have a comment here.
Or we just remove all parts here.